pyuvdata/uvdata/uvfits.py

```diff
@@ -197,6 +198,7 @@ def _get_parameter_data(
 
     def _get_data(
         self,
+        *,
         vis_hdu,
```

Comment:
I know they are more internal functions, but `_get_data` and `_get_parameter_data` have slightly different conventions. `_get_parameter_data` has `vis_hdu` as positional but this function requires keyword. I think we should just be consistent between the two.

Reply:
yes, I agree.